build/Targets/Roslyn.Toolsets.Xunit.targets

```diff
@@ -12,10 +12,39 @@
     <!-- xUnit.net WPF Runner -->
     <XUnitWpfPath>$(NuGetPackageRoot)\xunit.runner.wpf\$(xunitrunnerwpfVersion)\tools\xunit.runner.wpf.exe</XUnitWpfPath>
     <XUnitWpfArguments>"$(OutDir)\$(AssemblyName).dll"</XUnitWpfArguments>
-  </PropertyGroup>
 
-  <ItemGroup>
-    <PackageReference Include="xunit.runner.wpf" Version="$(xunitrunnerwpfVersion)" PrivateAssets="all" />
+    <AutoGenerateBindingRedirects>true</AutoGenerateBindingRedirects>
+    <ForceGenerationOfBindingRedirects>true</ForceGenerationOfBindingRedirects>
+
+    <RuntimeIdentifier Condition="'$(TargetFramework)' == 'net461' AND '$(RuntimeIdentifier)' == '' AND '$(OS)' == 'Windows_NT'">win7</RuntimeIdentifier>
+  </PropertyGroup>
+    
+  <ItemGroup Condition="'$(TargetFramework)' == 'netcoreapp2.0'">
+    <Content Include="$(NetStandard20RefPath)\netstandard.dll">
```

Comment:
Last time I tried upgrading to the latest netcoreapp2.0 they included System.ValueTuple and tons of tests that were testing error scenarios around not having System.ValueTuple started failing. How is that not the case now?

Reply:
The reason why is this is still the netstandard you were previously using for testing. Haven't changed this one yet. 

---
In reply to: [136669794](https://github.com/dotnet/roslyn/pull/21876#discussion_r136669794) [](ancestors = 136669794)